recipes/r-cpp11armadillo/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set version = '0.4.1' %}
+
+package:
+  name: r-cpp11armadillo
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/cpp11armadillo_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/cpp11armadillo/cpp11armadillo_{{ version }}.tar.gz
+  sha256: 826b8b0bb255b454704280cca34fe3720e2dc6ff39babd700609fbad3449d043
+
+build:
+  noarch: generic
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  host:
+    - r-base
+    - r-cpp11
+  run:
+    - r-base
+    - r-cpp11
+
+test:
+  commands:
+    - $R -e "library('cpp11armadillo')"
```

Bit of a nit as it's noarch, but the win only seems to fail because this is not conditional:

```suggestion
    - $R -e "library('cpp11armadillo')"           # [not win]
    - "\"%R%\" -e \"library('cpp11armadillo')\""  # [win]
```